### ISSUE TITLE
Documentation for SoftHSM is inaccessible

### ISSUE THREAD

**Issue by pete-fr** [softhsm/SoftHSMv2#737]
The [SoftHSM Page](https://www.opendnssec.org/softhsm/) contains the following link to the docs.

```
Documentation

How to install and use softHSM is described [here](https://wiki.opendnssec.org/display/SoftHSMDOCS).
```

This provides a login page with no link showing where to register, making the documentation inaccessible.

**Comment by pete-fr** [comment]
Apologies for raising an issue in github about this, I could not see another way to report this.

**Comment by bjosv** [comment]
Interesting, it seems that the wiki is a bit flaky.
I tried it successfully some week ago when commenting on #695, but I get the same problem as you now.

**Comment by bjosv** [comment]
FYI: It works fine again now when trying.

**Comment by welcor** [comment]
Still a problem. I get 

>     Not Found
>     The requested URL was not found on this server.

on both https://wiki.opendnssec.org/display/OpenDNSSEC and https://wiki.opendnssec.org/display/SoftHSMDOCS

**Comment by jschlyter** [comment]
Will be addressed in #765
